Sortable.js

```diff
@@ -1115,16 +1115,21 @@
 	/**
 	 * Returns the index of an element within its parent
 	 * @param el
+	 * @param selector
 	 * @returns {number}
 	 * @private
 	 */
-	function _index(/**HTMLElement*/el) {
+	function _index(/**HTMLElement*/el, /**String*/selector) {
 		if (!el || !el.parentNode) {
 			return -1;
 		}
 		var index = 0;
+		var className = typeof selector === 'undefined' || selector === '>*' ? ' ' : selector.replace('.', '');
```

Comment:
It will not work correctly, see the implementation in `_closest`.
Necessary to make the implementation of testing selector into a separate function (ex. `_matchSelector`) and use it.